### ISSUE TITLE
improvement: option to disable bmap on specific ranges

### ISSUE THREAD

**Issue by Louson** [yoctoproject/bmaptool#29]
Hello, 

Would it be possible to have an option specifying ranges for which all zeros would be copied ?

In my use case I have a wic image with a verity rootfs on the 3rd partition (let's say from bytes START to END). I would appreciate an option:

`bmaptool create --nomap START:END [...]`

so the mapping does not skip any zero between START and END.

**Comment by moto-timo** [comment]
The use cases are usually around bootloaders, so this impacts u-boot.env and pretty sure I had this problem on minnowboard EFI also (8 years ago or so vague memories).

**Comment by darko31** [comment]
Yes, flashing Mender images that have uboot.env file after the bootloader.
The uboot.env file is usually just empty 16MB filled with zeroes but there's a corner case where sdimg is flashed
with the bmap file and also rootfs is set to B partition. The newly flashed image will wipe the B partition but not the saved environment and u-boot will throw errors that it can not load device tree and kernel.

**Comment by schwesinger** [comment]
This is also relevant for #34 (for which I have a WIP patch set laying around, btw):  
When writing individual partitions to an existing destination image, the dest. file is not truncated. Unmapped blocks in the source file partition may, after overwriting the destination with it, then leave old, existing data intact.
